qemu/tests/virtio_serial_hot_unplug_port_pci.py

```diff
@@ -0,0 +1,76 @@
+import time
+import logging
+
+from virttest import error_context
+from virttest import utils_test
+from qemu.tests.virtio_serial_file_transfer import transfer_data
+from qemu.tests.hotplug_port_chardev_pci_with_console import get_virtio_serial_pci
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test hot unplug virtio serial devices.
+
+     1) Start guest with virtio serial device(s).
+     2) Transfer data via serial port.
+     3) Hot-unplug serial port
+     4) Hot-unplug chardev device
+     5) Hot-unplug virtio-serial-pci
+     6) Hotplug virtio-serial-pci
+     7) Hotplug chardev
+     8) Hotplug the port
+     9) Transfer data via port from guest to host
+     10) Hot-unplug serial the port
+     11) Hot-plug serial port to the same chardev.
+     12) Transfer data via the port from host to guest
+
+    :param test:   QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env:    Dictionary with test environment.
+    """
+    vm = env.get_vm(params["main_vm"])
+    unplug_pci = params.get("unplug_pci")
+    os_type = params["os_type"]
+    sleep_time = int(params.get("sleep_time"))
+    repeat_times = int(params.get("repeat_times"))
```

Comment:
Is there any other test cases use this script? Will `unplug_pci` and `repeat_times` have other values?

Reply:
Yes, when `unplug_pci = no` we won't hot-unplug pci. And the default value of `repeat_times` is 100.

Reply:
> Is there any other test cases use this script? Will `unplug_pci` and `repeat_times` have other values?

sorry, default value of  `repeat_times` is 1.

Reply:
> Is there any other test cases use this script? Will `unplug_pci` and `repeat_times` have other values?

Seems they are not the must so I delete them, thanks.